articles/libraries/lock-android/v2/custom-authentication-providers.md

```diff
@@ -3,17 +3,23 @@ section: libraries
 toc_title: Custom Authentication Providers
 description: Implementing custom authentication providers
 ---
-
 # Custom Authentication Providers
```

Comment:
This does not have a v1 version so it leads to the main doc https://auth0-docs-content-pr-5027.herokuapp.com/docs/libraries/lock-android/v1